consensus/hotstuff/verification/combined_signer_v2_test.go

```diff
@@ -30,21 +32,24 @@ func TestCombinedSignWithBeaconKey(t *testing.T) {
 	pk := beaconKey.PublicKey()
 	view := uint64(20)
```

Comment:
for clarity, I would rename this to 
```suggestion
	proposerView := uint64(20)
```